tests/test_search.py

```diff
@@ -1,17 +1,26 @@
 import unittest
 import requests
 from unittest.mock import patch, Mock
-from morphosource.search import search_media, get_media, Endpoints, ItemNotFound
+from morphosource.search import search_media, get_media, Endpoints, ItemNotFound, \
```

Comment:
```suggestion
from morphosource.search import search_media, get_media, Endpoints, ItemNotFound, 
```

Reply:
I get an error on the next line if I remove the backslash.